101-vm-sql-full-autobackup/azuredeploy.json

```diff
@@ -0,0 +1,484 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualMachineName": {
+      "type": "string",
+      "metadata": {
+        "description": "SQL Server Virtual Machine Name"
+      }
+    },
+    "virtualMachineSize": {
+      "type": "string",
+      "metadata": {
+        "description": "SQL Server Virtual Machine Size"
+      },
+      "allowedValues": [
+        "Standard_DS1",
+        "Standard_DS2",
+        "Standard_DS3",
+        "Standard_DS4",
+        "Standard_DS11",
+        "Standard_DS12",
+        "Standard_DS13",
+        "Standard_DS14"
+      ],
+      "defaultValue": "Standard_DS4"
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "SQL Server Virtual Machine Administrator User Name"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "SQL Server Virtual Machine Administrator Password"
+      }
+    },
+    "storageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "SQL Server Virtual Machine Storage Account Name"
+      }
+    },
+    "storageAccountType": {
+      "type": "string",
+      "metadata": {
+        "description": "SQL Server Virtual Machine Storage Account Type"
+      },
+      "allowedValues":[  
+            "Premium_LRS"
+         ],
+      "defaultValue":"Premium_LRS"
+    },
+    "virtualNetworkName": {
```

Comment:
Since we have the portal Single VM experience , this template is for Pro user where they can modify the template as much as they can so it is important to have all the parameters available. We have many customers waiting on this flexible solution.